File: C#Part2 - Homework/03. Methods - Homework/03.EnglishDigit/EnglishDigit.cs
namespace EnglishDigit
{
    using System;

    class EnglishDigit
    {
        // Write a method that returns the last digit of given integer as an English word.

        static void Main()
        {
            Console.WriteLine("Enter number");
            int num = int.Parse(Console.ReadLine());
            int lastDigit = num % 10;
            EnglishLastNum(lastDigit);
        }

        // Method
        static int EnglishLastNum(int number)
        {
            switch (number)
            {
                case 0: Console.WriteLine("Zero"); break;
                case 1: Console.WriteLine("One"); break;
                case 2: Console.WriteLine("Two"); break;
                case 3: Console.WriteLine("Three"); break;
                case 4: Console.WriteLine("Four"); break;
                case 5: Console.WriteLine("Five"); break;
                case 6: Console.WriteLine("Six"); break;
                case 7: Console.WriteLine("Seven"); break;
                case 8: Console.WriteLine("Eight"); break;
                case 9: Console.WriteLine("Nine"); break;
                default:
                    break;
            }
            return number;
        }
    }
}

File: JavaScript/Homework/Functions - Homework/Problem 1 English digit/EnglishDigit.js
// Write a function that returns the last digit of given integer as an English word.

var btn = document.getElementById('printNumber');

function returnLastDigit(number) {
    switch (number.toString()[number.length - 1] - '0') {
        case 0:
            return "zero";
            break;
        case 1:
            return "one";
            break;
        case 2:
            return "two";
            break;
        case 3:
            return "three";
            break;
        case 4:
            return "four";
            break;
        case 5:
            return "five";
            break;
        case 6:
            return "six";
            break;
        case 7:
            return "seven";
            break;
        case 8:
            return "eight";
            break;
        case 9:
            return "nine";
            break;
    }
}

btn.addEventListener('click', function () {
    var inputValue = document.getElementById('value').value;
    var resultArea = document.getElementById('result');
    result.innerHTML = "Last digit of value " + inputValue + " is " + returnLastDigit(inputValue)
})